### PR TITLE
tests: use HaveLen(...) and BeEmpty()

### PR DESCRIPTION
**Pull request by benukhanov**
Signed-off-by: Ben Oukhanov <boukhano@redhat.com>

**What this PR does / why we need it**:

Replace `Expect(len(...)).To(Equal(...))`
with `Expect(...).To(HaveLen(...))` and `Expect(...).To(BeEmpty())` to get
a better info when it fails.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

N/A

**Special notes for your reviewer**:

N/A

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/monitoring/vmistats/collector_test.go [hunk 1]
```diff
@@ -108,12 +108,12 @@ var _ = Describe("Utility functions", func() {
 
 			countMap = makeVMICountMetricMap(nil)
 			Expect(countMap).NotTo(BeNil())
-			Expect(len(countMap)).To(Equal(0))
+			Expect(countMap).To(HaveLen(0))
```
Comment (acardace) [inline pkg/monitoring/vmistats/collector_test.go:111]: I guess you probably wanted to use `BeEmpty()` here as well.

File: pkg/monitoring/vmistats/collector_test.go [hunk 2]
```diff
@@ -108,12 +108,12 @@ var _ = Describe("Utility functions", func() {
...
 
 			vmis := []*k6tv1.VirtualMachineInstance{}
 			countMap = makeVMICountMetricMap(vmis)
 			Expect(countMap).NotTo(BeNil())
-			Expect(len(countMap)).To(Equal(0))
+			Expect(countMap).To(HaveLen(0))
```
Comment (acardace) [inline pkg/monitoring/vmistats/collector_test.go:116]: Same as above for `BeEmpty()` instead of `HaveLen(0)`.

**Review by acardace — CHANGES_REQUESTED** [review]
Hi @benukhanov, thanks for the PR! I just have 2 tiny things.

File: pkg/monitoring/vmistats/collector_test.go (hunk 1 above)
Comment (EdDev) [reply pkg/monitoring/vmistats/collector_test.go:111]: `countMap` is a `map`, not `unit64`, so using `BeEmpty` makes sense.

Comment (benukhanov) [reply pkg/monitoring/vmistats/collector_test.go:111]: Ah, you're right. Thank you.

Comment (benukhanov) [reply pkg/monitoring/vmistats/collector_test.go]: Done.

File: pkg/monitoring/vmistats/collector_test.go (hunk 2 above)
Comment (benukhanov) [reply pkg/monitoring/vmistats/collector_test.go]: Done. Thanks.

**Review by EdDev — APPROVED** [review]
Thank you!

**Review by acardace — APPROVED** [review]

**Review by enp0s3 — APPROVED** [review]
/approve
/retest

**Pull request merged**